tools/docker/config/php.ini

```diff
@@ -11,7 +11,17 @@ error_log = /var/log/php/errors.log
 sendmail_path = /usr/sbin/ssmtp -t
 
 [XDebug]
-xdebug.mode = debug
+; General Xdebug config
 xdebug.client_host = host.docker.internal
 xdebug.client_port = 9003
 xdebug.log = /var/log/php/xdebug_remote.log
+xdebug.output_dir = /var/log/php
+xdebug.profiler_output_name = cachegrind.out.%p
+xdebug.mode = debug,profile
+xdebug.start_with_request = trigger
+
+; Trigger 'XDEBUG_TRIGGER=debug' for step debugging
+; Trigger 'XDEBUG_TRIGGER=profile' for profiler
+; Set different trigger values per mode
+xdebug.debug_trigger_value = debug
+xdebug.profiler_trigger_value = profile
```

Comment:
I'm not seeing any documentation about this?

As far as I can tell, with the configuration here it will activate `debug,profile` whenever XDEBUG_TRIGGER is set, no matter what the value. If you set `xdebug.trigger_value`, it will only activate `debug,profile` when XDEBUG_TRIGGER is set to the right value. But I don't see anything about controlling which modes are enabled via the value of XDEBUG_TRIGGER.

OTOH, it looks like (as long as you don't set `xdebug.trigger_value`) setting XDEBUG_PROFILE will enable only `profile` and XDEBUG_SESSION will enable only `debug`. This is described as "legacy" at https://xdebug.org/docs/all_settings#start_with_request#trigger though. 🤷